src/components/index.d.ts

```diff
@@ -383,7 +383,7 @@ export declare const filePath: (publicId: any) => any
 export declare const preloadImageData: ({
   crop, publicId, height, width,
 }: {
-  crop: keyof typeof CloudinaryImage.CROP_METHODS,
+  crop: string,
```

Comment:
To the person in the future that comes here, I was trying to do the right thing, but our current TS implementation doesn't support the `as const` keyword. `keyof typeof` here actually gives the type of the keys which is `FIT | FILL | CROP` and not the values, `fit | fill | crop`. `string` is just slightly better than what it was -- `any` 😄